### PR TITLE
fix: not disabled retry button when counting down

### PR DESCRIPTION
**Pull request by dawidgil**
Bug that i noticed after removing error with double running change detection

### CODE REVIEW

File: src/app/routes/applications/new-application/new-application.component.html [hunk 1]
```diff
@@ -40,16 +40,18 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
...
-            <div
-              class="card card-hint d-flex flex-row justify-content-start align-items-center shadow-none d-flex pt-3 pb-2 mb-3">
-              <mat-icon class="px-2">notifications_none</mat-icon>
-              <span class="pl-3 pb-2"> Please confirm this transaction in your connected Wallet. </span>
-            </div>
-            <div class="d-flex justify-content-end mb-3">
-              <button mat-stroked-button class="btn btn-small mr-2" (click)="closeDialog()">Cancel</button>
-              <button mat-raised-button class="btn btn-small btn-primary" (appRetryBtn)="retry()">Retry</button>
-            </div>
+            <ng-container *ngIf="stepper?.selectedIndex === i + 2">
```
Comment (jrhender) [inline src/app/routes/applications/new-application/new-application.component.html:43]: @dawidgil I don't quite get it. Why the `2` here?

Comment (dawidgil) [reply src/app/routes/applications/new-application/new-application.component.html:43]: We have 2 steps before ^^ First for describing values for application, second for summary. That's why there is 2. It's far from ideal, but i do not want to refactor to much, so i'm doing my best to adjust it.

**Comment by jrhender** [comment]
@dawidgil could you describe the repro steps for this bug please? I think that might help me understand the code change

**Comment by dawidgil** [comment]
Sure.
1. Go to governance -> create application.
2. Fill data for new application
3. Go through details forward and you will see retry button.
4. Accept metamask request signature.
5. You will go to the next step. 
Issue: The count down in retry button do not reset, so instead counting down from 60, it's continuing counting from previous step (for example you will have 30seconds). The farther you'll go, the less you will have in the clock.

**Review by whitneypurdum — APPROVED** [review]

**Pull request merged**